kf_lib_data_ingest/etl/transform/transform.py

```diff
@@ -109,9 +109,16 @@ def _write_output(self, output):
             for target_instance in data:
                 flat_instance = {}
                 for k, v in target_instance.items():
-                    if isinstance(v, dict):
+                    if k == 'properties':
                         flat_instance.update({k1: v1
                                               for k1, v1 in v.items()})
+                    elif k == 'links':
+                        for link_dict in v:
+                            flat_instance.update(
+                                {'source:' + k2: v2
+                                 for k2, v2 in link_dict.items()
+                                 if k2 != 'target_concept'}
+                            )
```

Comment:
What is this for?

Reply:
Is it for that other PR that changed the links format?

Reply:
Ah sorry yea, it should have been a separate commit